src/collective/solr/behaviors.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+from collective.solr import SolrMessageFactory as _
+from plone.autoform.interfaces import IFormFieldProvider
+from plone.supermodel import directives
+from plone.supermodel import model
+from zope import schema
+from zope.interface import provider
+
+
+@provider(IFormFieldProvider)
+class ISolrFields(model.Schema):
+    """ Social Media integration.
+    """
+
+    directives.fieldset(
+        'categorization',
+        fields=['showinsearch', 'searchwords']
+    )
+
+    showinsearch = schema.Bool(
+        required=False,
+        default=True,
+        missing_value=True,
+        title=_('label_showinsearch', default=u"Show in search"),
```

Comment:
Please do not mix, unless necessary, the quotation marks, Plone styleguide recommends single quotes whenever possible